p2p/raintree/addrbook_utils_test.go

```diff
@@ -161,6 +166,8 @@ func TestRainTreeAddrBookTargetsTwentySevenNodes(t *testing.T) {
 		{3, "X", "F"},
 		{2, "U", "["},
 		{1, "S", "W"},
+		{0, "X", "F"},  // redundancy
```

Comment:
Requests (for all the tree diagrams):
1. Could you update the spacing. I messed up in a few places
2. I think adding ** around where the expected raintree target is in the tree will help (e.g. around *U* and around *[*)
3. Could you add redundancy and cleanup via some visual format as well. I know we discussed this before, but even I forgot and am having a tough time trying to understand how it works.